src/Symfony/Component/Translation/Tests/Catalogue/DiffOperationTest.php

```diff
@@ -15,6 +15,9 @@
 use Symfony\Component\Translation\MessageCatalogue;
 use Symfony\Component\Translation\MessageCatalogueInterface;
 
+/**
+ * @group legacy
+ */
 class DiffOperationTest extends AbstractOperationTest
```

Comment:
What about extending the `TargetOperationTest` class instead and overriding only the `createOperation` method instead?